app/models/patient.rb

```diff
@@ -35,6 +35,7 @@ class Patient
   embeds_many :calls
   embeds_many :external_pledges
   embeds_many :notes
+  has_one 'current_user', class_name: 'User', inverse_of: :pledge_generated_by
```

Comment:
If memory serves, this adds a `current_user` attribute to patient objects, which isn't *quite* what we're looking for, I don't think. 

I think there's a good analog here in how we connect `clinic` to `patient` objects. We can get to the `clinic` object by doing `@patient.clinic`, but the attribute it's actually storing is `@patient.clinic_id` (it's just smart enough to do the lookup on its own).

I think that if we follow that pattern and alter it to a `belongs_to :pledge_generated_by`, it'll follow that same pattern and store an id nice and clean-like. And since there won't be an `inverse_of` to reference, we can just set that to `nil`.

short version: I think if we swap this to a `belongs_to :pledge_generated_by` with the same class name and an `inverse_of: nil`, I think this will blend beautifully and we'll be able to retrieve the user with a `@patient.pledge_generated_by`, nice and clean!